frontend/website2/pages/docs/my-first-transaction.mdx

```diff
@@ -76,13 +76,13 @@ Now we can access this everywhere as `$CODA_PK`. Note that this will only be sav
 
 In order to send our first transaction, we'll first need to get some coda to play with. Head over to the [Coda Discord server](https://bit.ly/CodaDiscord) and join the `#faucet` channel. Once there, ask Tiny the dog for some coda (you'll receive 100). Here's an example:
 
-    $request <public-key>
+    $request $<public-key>
```

Comment:
don't think we need the $ before the pk? In the discord, we just need the pk by itself (the $ is used in the env var when using shell eg. $CODA_PK)